src/signup.html

```diff
@@ -8,6 +8,15 @@
     <link rel="stylesheet" href="signup.css">
   </head>
   <body>
+
+    <!--breadcrumbs-->
```

Comment:
No need for comments like this when it's obvious from reading the code.

Usually comments are to add further context not apparent from reading the code.

```suggestion
```